sw/Modules/Comms/Src/can_sensors.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * can_sensors.c
+ *
+ *  Created on: Mar 19, 2025
+ *      Author: Hardy Yu
+ */
+
+#include "can_sensors.h"
+#include "spi.h"
+
+/* IMU static variables */
+extern SPI_HandleTypeDef hspi1;		// ASM330 SPI
+static uint32_t timestamp;
+
+/* time stamp mark*/
+static uint64_t next_10hz_service_at;
```

Comment:
I can't find this used anywhere else? (I just did a grep inside sw)
<img width="1382" alt="image" src="https://github.com/user-attachments/assets/a9fc1203-b022-4c63-a1b3-ca71235c4d8b" />


Also - the lack of space before `*/` bothers me a bit... :( idk why I noticed but now i can't unnotice